src/libmongoc/doc/learn/get/from-source.rst

```diff
@@ -0,0 +1,376 @@
+###########################################
+Building the C Driver Libraries from Source
+###########################################
+
+.. highlight:: shell-session
+.. default-role:: bash
+
+To build the C driver from source, one first needs to obtain the source code.
+The easiest way to obtain the source is to download it from
+`the official GitHub repository`__.
+
+__ https://github.com/mongodb/mongo-c-driver
+
+.. note::
+
+  In the following examples, `$SOURCE_DIR` refers to the directory in which you
+  will be placing the |mongo-c-driver| source code, and `$VERSION` refers to the
+  version of the C driver that you are. The current version written for this
+  documentation is |version.pre|.
+
+It is **highly recommended** that new users use a stable released version of the
+driver, rather than building from a development branch.
```

Comment:
Suggest using an "important" admonition here and moving it up above the note for priority.

Similarly, recommend use of the "important" admonition for other "highly recommended" remarks.